src/app/handlers/blacklist.handler.ts

```diff
@@ -22,13 +27,16 @@ export class BlacklistHandler implements IHandler {
       return;
     }
 
-    this._expressions.forEach(({regex, label}) => {
+    this._expressions.forEach(({ regex, label }) => {
       if (message.content.toLowerCase().match(regex)) {
+        message.author.send(
+          `Please do not share \`${label}\` links in the \`${message.guild.name}\` server.`
```

Comment:
use regular `'` to avoid escaping

Reply:
The back ticks were there for formatting.  Not sure if that is what you are referring too.

Reply:
nvm, im confused